packages/matchbox/src/components/OptionalLabel/OptionalLabel.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Box } from '../Box';
+
+const StyledBox = styled(Box)`
+  float: right;
+`;
+
+function OptionalLabel() {
+  return (
+    <StyledBox as="span" fontSize="200" fontWeight="light" lineHeight="200" m="0" color="gray.700">
```

Comment:
I think this is supposed to be normal font weight https://sparkpost.invisionapp.com/overview/optionalStyling-ckb27k05a00iu012xgf3nhz6r/screens